spec/unit/core_spec.rb

```diff
@@ -123,4 +123,28 @@ class Book < ActiveFedora::Base
       it { should eq '123456w' }
     end
   end
+
+  describe "with an empty base_path" do 
+    let(:book) { Book.create(title:"foo") }
```

Comment:
This can be factored out into a describe block that contains all three of the blocks you added. Also factor out:
```
allow(ActiveFedora.fedora).to receive(:base_path).and_return(base)
```
Each block can `let(:base) { '/' }` as appropriate